### DISCUSSION TITLE
Trigger Python Scripts

### DISCUSSION THREAD

**Discussion by rowalth** [Budibase/budibase#9186] [Help]
I am a complete newbie, so please forgive me obvious questions. Is it possible to trigger python scripts that manipulate the data that I pulled from google sheets source? If yes, who do I approach this & where do I find the specific information?

**Comment by melohagan** [comment]
As mentioned on discord:

> You could potentially run some python on an AWS lambda or other flask service, and then use the REST connector in Budibase to POST data to it
